e2e/cli_test.go

```diff
@@ -182,6 +183,31 @@ func TestLintAllViolations(t *testing.T) {
 	}
 }
 
+func TestTestRegalBundledRules(t *testing.T) {
+	t.Parallel()
+
+	stdout := bytes.Buffer{}
+	stderr := bytes.Buffer{}
```

Comment:
Not for this PR, just a random thought -- I wonder if something like this would be useful for testing:

```go
type panicWriter struct{}

func (*panicWriter) Write([]byte) (int, error) {
	panic("write unexpected")
}
```

It's like `io.Discard` just with a little more drama. https://play.golang.com/p/xCQlGZH2nUq

Reply:
Yeah you've made a good point there. Perhaps something that takes a `*testing.T` and fails accordingly would strike a middle ground... maybe some day 🙃